sources/packages/backend/libs/utilities/src/esdc-utils.ts

```diff
@@ -24,10 +24,14 @@ export function getMaritalStatusCode(
  * Gets the gender code conversion for the ESDC request files.
  */
 export function getGenderCode(gender: string): string {
-  if (gender === "male") {
+  if (gender === "man") {
```

Comment:
As a suggestion, a switch could be used instead.

Reply:
update